### ISSUE TITLE
It need to read onTapHint, onLongPressHint and other hint texts by VoiceOver

### ISSUE THREAD

**Issue by yama3nomori** [flutter/flutter#69352]
It is written following:

> ?On iOS, these values are always ignored.
<a href="https://api.flutter.dev/flutter/semantics/SemanticsHintOverrides-class.html">https://api.flutter.dev/flutter/semantics/SemanticsHintOverrides-class.html</a>

But this information is very important for VoiceOver users.

I'm feeling the accessibility of iOS apps is more insufficient than Android apps_built by Flutter.
Flutter is very good flamework.
I ask please be also best accessibility features of iOS.

**Comment by jonahwilliams** [comment]
iOS does not provide the capability to override these hints. There is still a default hint for tappable and long-pressable elements, it is just controlled entirely by the OS

**Comment by darshankawar** [comment]
@yama3nomori 
Based on Jonah's comment above, I am going ahead and closing this as-designed.
